docs/docs/k8s/ingress.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Ingress Controller
+lang: en-US
+sidebarDepth: 1
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy oidc kubernetes Ingress reverse-proxy
+---
+
+# Kubernetes Ingress Controller
+
+Use Pomerium as a first-class secure-by-default Ingress Controller. Dynamically provision routes from Ingress resources and set policy based on annotations. By defining routes as Ingress resources you can independently create and remove them from Pomerium's configuration. This enables workflows more native to Kubernetes environments such as actions based on pull requests.
+
+## Prerequisites
+
+- A certificate management solution. If you do not already have one in place, this article covers using Cert Manager.
+- A Redis backend with high-persistence is highly recommended.
+
+### System Requirements
+
+- Kubernetes v1.19.0+
+- Pomerium [Helm Chart](https://github.com/pomerium/pomerium-helm/tree/master/charts/pomerium) v25.0.0+
+
+### Limitations
+
+::: warning
+
+Only one Ingress Controller instance/replica is supported per Pomerium cluster.
+
+:::
+
+## Installation
+
+### Helm
+
+Our instructions for [Installing Pomerium Using Helm](/docs/k8s/helm.md) includes the Ingress Controller as part of the documented configuration.
+
+
+```yaml
+ingressController:
+  enabled: true
+```
+
+### Docker Image
+
+You may deploy your own manifestations by using the `pomerium/ingress-controller` docker image.
+
+## Configuration
+
+|  Flag                          | Description                                                          |
+| ------------------------------ | -------------------------------------------------------------------- |
+| `--databroker-service-url`     | the databroker service url
+| `--databroker-tls-ca`          | base64 encoded tls CA
+| `--databroker-tls-ca-file`     | tls CA file path for the databroker connection connection
+| `--health-probe-bind-address`  | The address the probe endpoint binds to. (default ":8081")
+| `--metrics-bind-address`       | The address the metric endpoint binds to. (default ":8080")
+| `--name`                       | IngressClass controller name (default "pomerium.io/ingress-controller")
+| `--namespaces`                 | namespaces to watch, or none to watch all namespaces
```

Comment:
is `none` a value?

Reply:
Yes, updated.

Reply:
`none` is not literal, it just means omitting this option

Reply:
OK, changing to:

> Namespaces to watch, omit to watch all namespaces

Thanks.

Reply:
Omitting it would imply none.  In usage it is just the list of namespaces to watch; defaulting to all.